dom.bs

```diff
@@ -4982,6 +4982,13 @@ invoked, must run these steps:
     <tr><td>"<code>wheelevent</code>"<td>{{WheelEvent}}<td>[[!UIEVENTS]]
   </table>
 
+ <li>
+  If the user agent does not support the <var>constructor</var> interface, or has it disabled
+  in its current configuration, let <var>constructor</var> be null.
+
+  <p class=note>Typically user agents have the {{TouchEvent}} interface disabled in some
+  configurations.
```

Comment:
Yes but that would result in a `TypeError` whereas WebKit/Chromium/Gecko (at last) throw `NotSupportedError`.
http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=4078

I suppose `TypeError` is Web compatible enough but are implementors OK with changing to `TypeError`?